docs/user-guide/configuration-files/strongbox-security-users-yaml.md

```diff
@@ -1,6 +1,6 @@
 # The `strongbox-security-users.yaml` File
 
-TODO @sbespalov
+
```

Comment:
The `TODO`s are here for a reason. If you remove them, we won't know we have a `TODO` as it won't come up in the IDEs. :)

Reply:
I removed them due to them showing up in the wiki. It shows that there are todo's for these items. I will revert these changes to add them back in.